raster/r.carve/r.carve.html

```diff
@@ -3,7 +3,15 @@ <h2>DESCRIPTION</h2>
 <em>r.carve</em> accepts vector stream data as input, transforms them to
 raster, and subtracts a default-depth + additional-depth from a DEM. If
 the given width is more than 1 cell, it will carve the stream with the
-given width. With the <b>-n</b> flag it should eliminate all flat cells within
+given width. It is possible to define different width and depth (column
+in the vector map table) separately for each line segment. Another way is to
+combine the value from the vector map table column (represent width or depth)
+and the width or depth parameter. If the width or depth value in the
```

Comment:
```suggestion
and the <b>width</b> or <b>depth</b> parameter. If the width or depth value in the
```
(parameter names in bold, see https://trac.osgeo.org/grass/wiki/Submitting/Docs#Markupstyleguide)

Reply:
Thanks Markus. I've already fixed it.